lib/phoenix_live_view/test/dom.ex

```diff
@@ -44,17 +44,19 @@ defmodule Phoenix.LiveViewTest.DOM do
     case Floki.attribute(node, "id") do
       [id] ->
         if MapSet.member?(ids, id) do
-          raise """
-          Duplicate id found: #{id}
+          IO.warn("""
+          Duplicate id found while testing LiveView: #{id}
+
+          #{inspect_html(node)}
 
           LiveView requires that all elements have unique ids, duplicate IDs will cause
           undefined behavior at runtime, as DOM patching will not be able to target the correct
           elements.
-          """
-        else
-          detect_duplicate_ids(children, MapSet.put(ids, id))
+          """)
```

Comment:
We probably want to pass `[]` as second argument? We don't have a useful stacktrace here, do we?

Reply:
sadly no good stacktrace. Changed!

Reply:
actually, at least it contains the test, so maybe we should keep it?

```
warning: Duplicate id found while testing LiveView: foo

    <div id="foo"></div>


LiveView requires that all elements have unique ids, duplicate IDs will cause
undefined behavior at runtime, as DOM patching will not be able to target the correct
elements.

  (phoenix_live_view 1.0.1) lib/phoenix_live_view/test/dom.ex:47: Phoenix.LiveViewTest.DOM.detect_duplicate_ids/2
  (phoenix_live_view 1.0.1) lib/phoenix_live_view/test/dom.ex:36: Phoenix.LiveViewTest.DOM.detect_duplicate_ids/2
  (phoenix_live_view 1.0.1) lib/phoenix_live_view/test/dom.ex:28: Phoenix.LiveViewTest.DOM.parse/1
  test/phoenix_live_view/test/dom_warn_test.exs:10: Phoenix.LiveViewTest.DOMWarnTest."test parse detects duplicate ids"/1
  (ex_unit 1.19.0-dev) lib/ex_unit/runner.ex:511: ExUnit.Runner.exec_test/2
  (stdlib 6.1.2) timer.erl:590: :timer.tc/2
  (ex_unit 1.19.0-dev) lib/ex_unit/runner.ex:433: anonymous fn/6 in ExUnit.Runner.spawn_test_monitor/4
```

Reply:
Yeah, we keep it!